File: javascript/Leetcode/problem list/66plusOne.py
class Solution(object):
    def plusOne(digits):
        total = 0
        stack = []
        for i in range(len(digits)):
            total += digits[i] * (10 ** (len(digits) - i - 1))
        for digit in str(total + 1):  
            stack.append(int(digit))
        return stack

File: javascript/Leetcode/problem list/66plusOne.js
function plusOne(digits) {
    
    let sum = 0;
    let stack = []
    for (let i = 0; i < digits.length; i++) {
        sum = sum + digits[i]*(10**(digits.length -i-1))
        console.log(sum);
        
    }
    
    String(sum).split("").forEach(i=>{
        stack.push(BigInt(i))
    })

    return stack
}
console.log(plusOne([6,1,4,5,3,9,0,1,9,5,1,8,6,7,0,5,5,4,3]));
